--- origsrc/ocaml-sqlite-0.3.5/Makefile.in	2004-04-14 16:35:05.000000000 -0500
+++ src/ocaml-sqlite-0.3.5/Makefile.in	2008-01-08 22:30:57.296875000 -0600
@@ -35,7 +35,6 @@
 # one in . 
 install: sqlite.cma sqlite.cmxa sqlitetop
 	ocamlfind install sqlite META sqlite.cmi sqlite.mli sqlite.cma *.cmxa *.a *.cmx *.so
-	ocamlfind ocamlmktop -o sqlitetop -package "sqlite" sqlite.cma
 	$(INSTALL) sqlitetop $(bindir)/sqlitetop
 
 remove: uninstall
